src/Microsoft.ML.Ensemble/Trainer/EnsembleModelParametersBase.cs

```diff
@@ -13,8 +13,8 @@
 
 namespace Microsoft.ML.Runtime.Ensemble
 {
-    public abstract class EnsemblePredictorBase<TPredictor, TOutput> : PredictorBase<TOutput>,
-        IPredictorProducing<TOutput>, ICanSaveInTextFormat, ICanSaveModel, ICanSaveSummary
+    public abstract class EnsembleModelParametersBase<TPredictor, TOutput> : ModelParametersBase<TOutput>,
```

Comment:
>public [](start = 4, length = 6)

:)

Reply:
Making this internal as it is the base class

---
In reply to: [242426043](https://github.com/dotnet/machinelearning/pull/1901#discussion_r242426043) [](ancestors = 242426043)

Reply:
Doc strings

---
In reply to: [243467555](https://github.com/dotnet/machinelearning/pull/1901#discussion_r243467555) [](ancestors = 243467555,242426043)